docs/developers/contributing/documentation/docs_deployment.md

```diff
@@ -16,36 +16,38 @@ through several CI workflows detailed below.
     - **Workflow file:** [`build_docs.yml`](https://github.com/napari/napari/blob/main/.github/workflows/build_docs.yml)
         - **job:** `build-and-upload`
         - Pulls in sources from `napari/docs` and builds docs locally. Uploads
-          artifacts to this repo (`napari/napari`).
-        - This is triggered on every Pull Request and shows up as a "Build PR
-          Docs" check on the PR.
+          artifacts to this repository (`napari/napari`).
+        - This is triggered on any push to a branch named 'docs*' or tag named 'v*',
+          to `napari/napari`. The artifact can be used to check the documentation
+          but is not used for docs deployment.
     - **Workflow file:** [`deploy_docs.yml`](https://github.com/napari/napari/blob/main/.github/workflows/deploy_docs.yml)
-        - **job:** `build-and-deploy`
-        - Triggers [`deploy_docs.yml`](https://github.com/napari/docs/blob/main/.github/workflows/deploy_docs.yml)
+        - **job:** `build-napari-docs`
+        - Triggers [`build_and_deploy.yml`](https://github.com/napari/docs/blob/main/.github/workflows/build_and_deploy.yml)
           workflow at the [napari/docs](https://github.com/napari/docs) repo.
           Waits for results and reports it.
         - This is triggered on any commit to the `main` branch on
           `napari/napari` (and consequently triggers a new deployment of the
-          `napari.org` website.)
+          `napari.org` website). When the commit is tagged, the `build_and_deploy.yml`
+          workflow will deploy to the version folder e.g., `0.4.19/`.
```

Comment:
Do we want to use some version/variable substitution here?
I think `{{ napari_version }}` would use the version of the release?

Reply:
Done!

Reply:
It didnt work:

![image](https://github.com/napari/docs/assets/23182829/a5abbc56-de4a-4b0b-88bb-10a170056b15)

Reply:
Nevermind, I had backticks. It works!